ibexa/product-catalog-date-time-attribute/4.6/config/ibexa_product-catalog-date-time-attribute.yaml

```
@@ -0,0 +1,3 @@
+# Example configuration for a custom date and time attribute:
+
+#ibexa_product-catalog-date-time-attribute:
```

Also file name needs to be adjusted. Besides, I would provide some example configuration, similar to https://github.com/ibexa/recipes-dev/blob/master/ibexa/product-catalog-symbol-attribute/4.6/config/ibexa_product_catalog_symbol_attribute.yaml.
```suggestion
#ibexa_product_catalog_date_time_attribute:
```